pom.xml

```diff
@@ -1228,6 +1228,7 @@
           <consoleOutput>true</consoleOutput>
           <reportFile>${project.build.directory}/${project.build.finalName}.rat</reportFile>
           <excludes>
+            <exclude>**/connectors/**/*.properties</exclude>
```

Comment:
Why exclude this module?  Just fill-in correct licenses in mentioned files during build.